src/lightly_train/_commands/train_task_helpers.py

```diff
@@ -650,6 +650,7 @@ def load_checkpoint_from_file(
     fabric: Fabric,
     ckpt_path: PathLike,
     state: TrainTaskState,
+    reuse_class_head: bool = False,
```

Comment:
```suggestion
    reuse_class_head: bool,
```
For these internal functions I would not set defaults unless necessary.